src/sinol_make/helpers/package_util.py

```diff
@@ -0,0 +1,33 @@
+import os
+
+
+def get_task_id() -> str:
+    return os.path.split(os.getcwd())[-1]
+
+
+def extract_test_no(test_path):
+    return os.path.split(os.path.splitext(test_path)[0])[1][3:]
```

Comment:
Some comments with examples would be helpful to improve the maintainability of the code. Currently I have to try to deduce what it does based on the function name and compare it with my deduction of what the code does.

Reply:
Also, is it a `no` (number), or a string?

Reply:
It's a string, for "in/abc1a.in" it returns "1a". But I don't know how to make this function's name better.

Reply:
`extract_test_name_without_tag`, or `extract_test_id`? We can also add comments with examples so that it's clear.

Reply:
I changed the name 9c22ee1cb16d05dbc6dd62757dc415656892606c